yarn-project/acir-simulator/src/client/view_data_oracle.ts

```diff
@@ -57,7 +57,16 @@ export class ViewDataOracle extends TypedOracle {
   }
 
   /**
-   * Gets some notes for a storage slot.
+   * Pops a capsule from the capsule dispenser
+   * @returns The capsule values
+   */
+  public popCapsule(): Promise<Fr[]> {
+    return this.db.popCapsule();
+  }
+
+  /**
+   * Gets some notes for a contract address and storage slot.
+   * Returns a flattened array containing real-note-count and note preimages.
```

Comment:
```suggestion
   * Returns a flattened array containing filtered notes.
```

Why the "real-note-count"? It just returns an array of notes.